src/Idea.hs

```diff
@@ -90,16 +90,30 @@ idea severity hint from to = rawIdea severity hint (srcInfoSpan $ ann from) (f f
 idea' :: (GHC.HasSrcSpan a, Outputable.Outputable a) =>
          Severity -> String -> a -> a -> [Refactoring R.SrcSpan] -> Idea
 idea' severity hint from to =
-  rawIdea severity hint (ghcSpanToHSE (GHC.getLoc from)) (f from) (Just $ f to) []
-  where
-    f = Outputable.showSDocUnsafe . Outputable.ppr
+  rawIdea severity hint (ghcSpanToHSE (GHC.getLoc from)) (unsafePrettyPrint from) (Just $ unsafePrettyPrint to) []
 
 suggest = idea Suggestion
 warn = idea Warning
+
+ignoreNoSuggestion' :: (GHC.HasSrcSpan a, Outputable.Outputable a)
+                    => String -> a -> Idea
+ignoreNoSuggestion' hint x = rawIdeaN Ignore hint (ghcSpanToHSE (GHC.getLoc x)) (unsafePrettyPrint x) Nothing []
```

Comment:
I think this is better than directly calling `rawIdeaN` with arguments, but it's only currently used in `NewType.hs`. Is this fine? (naming?)

Reply:
It's fine - none of the names here are good, so we can refactor wholesale if we think of something better.